pkg/inspector/image-inspector.go

```diff
@@ -154,15 +154,17 @@ func (i *defaultImageInspector) Inspect() error {
 		if err != nil {
 			i.meta.OpenSCAP.SetError(err)
 			log.Printf("DEBUG: Unable to scan image %q with OpenSCAP: %v", i.opts.Image, err)
+			return err
 		} else {
 			i.meta.OpenSCAP.Status = iiapi.StatusSuccess
-		}
 
-		report := reportObj.(openscap.OpenSCAPReport)
-		scanReport = report.ArfBytes
-		htmlScanReport = report.HTMLBytes
+			report := reportObj.(openscap.OpenSCAPReport)
+			scanReport = report.ArfBytes
+			htmlScanReport = report.HTMLBytes
+
+			scanResults.Results = append(scanResults.Results, results...)
```

Comment:
yep; when that gets merged i'll rebase this. i can remove it from the committed changes here until then, but it's annoying to get panics when trying to test error cases